xnat_admin_tools/create_new_projects.py

```diff
@@ -50,6 +50,96 @@ def remove_empty(values_to_select, project_values):
     return values_to_insert
 
 
+def set_xsync_credentials(
+    xrelay_host: str,
+    xrelay_user: str,
+    xrelay_pass: str,
+    xserver_host: str,
+    xserver_user: str,
+    xserver_pass: str,
+    project_id: str,
+):
+    """
+    Sets remote server credentials for Xsync service on the relay
+
+    If this step fails please set credentials manually
+    """
+    import json
+
+    import requests
+    from requests.auth import HTTPBasicAuth
+
+    basic = HTTPBasicAuth(xrelay_user, xrelay_pass)
+    payload = {
+        "username": xserver_user,
+        "secret": xserver_pass,
+        "host": xserver_host,
+        "alias": xserver_user,
+        "localProject": project_id,
+        "remoteProject": project_id,
+    }
+
+    post_url = xrelay_host + "/xapi/xsync/credentials/save/projects/" + project_id
+    # print(post_url)
+    response = requests.request(
+        "POST",
+        post_url,
+        headers={"Content-Type": "text/plain"},
+        data=json.dumps(payload),
+        auth=basic,
+    )
+
+    return response
+
+
+def set_project_settings(
+    xrelay_host: str,
+    xrelay_user: str,
+    xrelay_pass: str,
+    xserver_host: str,
+    project_id: str,
+):
+    """
+    Sets project settings for the project on the relay
+
+    If this step fails please set the project settings manually
+    """
+    import json
+
+    import requests
+    from requests.auth import HTTPBasicAuth
+
+    basic = HTTPBasicAuth(xrelay_user, xrelay_pass)
+    payload = {
+        "project_resources": {"sync_type": "none"},
+        "subject_resources": {"sync_type": "none"},
+        "subject_assessors": {"sync_type": "none"},
+        "imaging_sessions": {"sync_type": "all"},
+        "enabled": True,
+        "source_project_id": project_id,
+        "sync_frequency": "on demand",
+        "sync_new_only": True,
+        "identifiers": "use_local",
+        "remote_url": xserver_host,
+        "remote_project_id": project_id,
+        "notification_emails": "",
```

Comment:
I wonder if we should try to notify the users?

Reply:
I guess we're keeping this on hold based on the conversation in last BNC meeting